### ISSUE TITLE
[Blue Burst] Error 100 while sorting Guild Cards in the same position

### ISSUE THREAD

**Issue by nolrinale** [fuzziqersoftware/newserv#312]
Receive a guild card from a friend, then press F5 and go to Card List, select the card and press "Sort", sort it manually into the same spot it was before, you will get an instant Error 100 with the following error:

```
Jan 02 20:22:47 kiyomizu.moe newserv[158286]: I 158286 2024-01-02 20:22:47 - Cleared 1 expired system file(s)
Jan 02 20:22:22 kiyomizu.moe newserv[158286]: I 158286 2024-01-02 20:22:22 - [C-6] Deleted
Jan 02 20:22:22 kiyomizu.moe newserv[158286]: I 158286 2024-01-02 20:22:22 - Saved Guild Card file system/players/guild_cards_myria.psocard
Jan 02 20:22:22 kiyomizu.moe newserv[158286]: I 158286 2024-01-02 20:22:22 - Saved character file system/players/player_myria_1.psochar
Jan 02 20:22:22 kiyomizu.moe newserv[158286]: I 158286 2024-01-02 20:22:22 - Added 3 seconds to play time
Jan 02 20:22:22 kiyomizu.moe newserv[158286]: I 158286 2024-01-02 20:22:22 - Saved system file system/players/system_myria.psosys
Jan 02 20:22:22 kiyomizu.moe newserv[158286]: I 158286 2024-01-02 20:22:22 - [Server] Client disconnected: C-6 on fd 37
Jan 02 20:22:22 kiyomizu.moe newserv[158286]: W 158286 2024-01-02 20:22:22 - [Server] Error processing client command: invalid new position
Jan 02 20:22:22 kiyomizu.moe newserv[158286]: 0000 | 10 00 E8 0A 00 00 00 00 8B 07 BC C2 8B 07 BC C2 |
Jan 02 20:22:22 kiyomizu.moe newserv[158286]: I 158286 2024-01-02 20:22:22 - [Commands] Received from C-6 (TOUKA) (version=BB command=0AE8 flag=00000000)
Jan 02 20:22:22 kiyomizu.moe newserv[158286]: 0010 | 2D FA 3D C3                                     | - =
Jan 02 20:22:22 kiyomizu.moe newserv[158286]: 0000 | 14 00 60 00 00 00 00 00 42 03 00 00 D4 6B C7 C2 |   `     B    k
Jan 02 20:22:22 kiyomizu.moe newserv[158286]: I 158286 2024-01-02 20:22:22 - [Commands] Received from C-6 (TOUKA) (version=BB command=0060 flag=00000000)
Jan 02 20:22:22 kiyomizu.moe newserv[158286]: 0010 | EF E4 39 C3                                     |   9
Jan 02 20:22:22 kiyomizu.moe newserv[158286]: 0000 | 14 00 60 00 00 00 00 00 42 03 00 00 5F 41 D4 C2 |   `     B   _A
Jan 02 20:22:22 kiyomizu.moe newserv[158286]: I 158286 2024-01-02 20:22:22 - [Commands] Received from C-6 (TOUKA) (version=BB command=0060 flag=00000000)
Jan 02 20:22:22 kiyomizu.moe newserv[158286]: 0010 | 84 C3 3A C3                                     |   :
Jan 02 20:22:22 kiyomizu.moe newserv[158286]: 0000 | 14 00 60 00 00 00 00 00 42 03 00 00 79 27 E3 C2 |   `     B   y'
Jan 02 20:22:22 kiyomizu.moe newserv[158286]: I 158286 2024-01-02 20:22:22 - [Commands] Received from C-6 (TOUKA) (version=BB command=0060 flag=00000000)
Jan 02 20:22:22 kiyomizu.moe newserv[158286]: 0010 | C8 A1 3B C3                                     |   ;
Jan 02 20:22:22 kiyomizu.moe newserv[158286]: 0000 | 14 00 60 00 00 00 00 00 42 03 00 00 A9 0D F2 C2 |   `     B
Jan 02 20:22:22 kiyomizu.moe newserv[158286]: I 158286 2024-01-02 20:22:22 - [Commands] Received from C-6 (TOUKA) (version=BB command=0060 flag=00000000)
Jan 02 20:22:22 kiyomizu.moe newserv[158286]: 0010 | 2A 7F 3C C3                                     | * <
Jan 02 20:22:22 kiyomizu.moe newserv[158286]: 0000 | 14 00 60 00 00 00 00 00 42 03 00 00 06 7A 00 C3 |   `     B    z
Jan 02 20:22:22 kiyomizu.moe newserv[158286]: I 158286 2024-01-02 20:22:22 - [Commands] Received from C-6 (TOUKA) (version=BB command=0060 flag=00000000)
Jan 02 20:22:21 kiyomizu.moe newserv[158286]: 0010 | 7F 5A 3D C3                                     |  Z=
Jan 02 20:22:21 kiyomizu.moe newserv[158286]: 0000 | 14 00 60 00 00 00 00 00 42 03 00 00 70 ED 07 C3 |   `     B   p
Jan 02 20:22:21 kiyomizu.moe newserv[158286]: I 158286 2024-01-02 20:22:21 - [Commands] Received from C-6 (TOUKA) (version=BB command=0060 flag=00000000)
Jan 02 20:22:21 kiyomizu.moe newserv[158286]: 0010 | DD A8 3E C3                                     |   >
Jan 02 20:22:21 kiyomizu.moe newserv[158286]: 0000 | 14 00 60 00 00 00 00 00 42 03 00 00 56 43 0F C3 |   `     B   VC
Jan 02 20:22:21 kiyomizu.moe newserv[158286]: I 158286 2024-01-02 20:22:21 - [Commands] Received from C-6 (TOUKA) (version=BB command=0060 flag=00000000)
Jan 02 20:22:21 kiyomizu.moe newserv[158286]: 0010 | E9 D0 3E C3                                     |   >
```

**Comment by fuzziqersoftware** [comment]
This log is kind of confusing because it's presented in reverse. What log system/viewer are you using?
Anyway, this issue is fixed by fccc0f73.

**Comment by nolrinale** [comment]
Oh I was using journalctl for this one and I passed the command `journalctl -u newserv -r` to get it when it happened.

Without the -r it's output is the usual but i had to scroll a big chunk to get to the specific line
